src/types.ts

```diff
@@ -1709,6 +1710,11 @@ export type PartialUserUpdate<UserType = UnknownType> = {
   unset?: Array<keyof UserResponse<UserType>>;
 };
 
+export type PartialMessageUpdate<MessageType = UnknownType> = {
+  set?: Partial<MessageResponse<MessageType>>;
+  unset?: Array<keyof MessageResponse<MessageType>>;
```

Comment:
do we allow updates on all the `MessageResponse` properties, like `message.cid`?

Reply:
no, these are the allowed fields:

- custom data (any field)
- text
- html (server side)
- attachments
- mentioned_user_ids
- quoted_message_id
- silent
- pin